TimeAlarms.cpp

```diff
@@ -66,6 +66,37 @@ void AlarmClass::updateNextTrigger()
           // set the date to this week today and add the time given in value
           nextTrigger = value + previousSunday(time);
         }
+      }
+      else if (Mode.alarmType == dtMultiWeeklyAlarm) {
+        // if this is a multi weekly alarm
+        boolean updated = false;
+
+        for (int dayIndex = (dowSunday-1); dayIndex != (dowSaturday-1); dayIndex++) {
```

Comment:
This loop is missing Friday and Saturday, is going from (0 to 5)

```c++
for (int dayIndex = (dowSunday); dayIndex <= (dowSaturday); dayIndex++) {
```

Reply:
Thank you for your feedback. I just committed a fix.
And sorry for the delay